### PR TITLE
SwaggerUI validation error after Heroku deployment

### PR DESCRIPTION
**Pull request by chrispascucci**
Added options to app.js to set the validation url in the swagger-ui-express package.

### CODE REVIEW

**Comment by BWortman** [comment]
Something doesn't look quite right. There's a little image placeholder on the screen:

image

And clicking it takes you to an odd looking page:

image

**Comment by rclanan** [comment]
In local mode, it needs to be set to null since it can't reach the validate anyways. That's the default behavior for the validator. Once it's deployed we can check to see if that actually worked for the URL but I though it was looking for an actual URL.

**Comment by rclanan** [comment]
When it's deployed, the Url needs to be set to "https://autorenter-nodeexpress-api.herokuapp.com/swagger.json"

We have a validation error as well: https://online.swagger.io/validator/debug?url=https://autorenter-nodeexpress-api.herokuapp.com/swagger.json

{"schemaValidationMessages":[{"level":"error","domain":"validation","keyword":"anyOf","message":"instance failed to match at least one required schema among 2","schema":{"loadingURI":"http://json-schema.org/draft-04/schema#","pointer":"/properties/type"},"instance":{"pointer":"/definitions/Vehicle/properties/image/type"}}]}

File: server/app.js [hunk 1]
```diff
@@ -46,7 +46,11 @@ app.get('/swagger.json', function(request, response) {
   response.json(swaggerSpec);
 });
 
-app.use('/docs/api', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
+const options = {
+  validatorUrl: './swagger.json'
```
Comment (rclanan) [inline server/app.js]: You should add a check here to see if running it in dev or production mode, might have to pass in a command line argument or something to achieve that. 

validatorUrl: isProd ? 'https://autorenter-nodeexpress-api.herokuapp.com/swagger.json' : null

File: server/app.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const swaggerJSDoc = require('swagger-jsdoc');
 const config = require('./config');
 
 const app = module.exports = express();
-
+const env = app.get('env');
```
Comment (rclanan) [inline server/app.js]: Check how this is done in the config file. Shouldn't need to introduce this here. 

https://github.com/fusionalliance/autorenter-nodeexpress-api/blob/development/server/config.js#L8

File: server/app.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ app.get('/swagger.json', function(request, response) {
 });
 
 const options = {
-  validatorUrl: './swagger.json'
+  validatorUrl: env === 'production' ? 'https://autorenter-nodeexpress-api.herokuapp.com/swagger.json' : null //eslint-disable-line
```
Comment (rclanan) [inline server/app.js]: So this should end up like config.validationUrl

**Pull request merged**